src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,96 @@
+.nav {
+  display: flex;
+  height: 60px;
+  padding: 0 50px;
+  background-color: #fff;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 2px 4px $black;
+
+  &__logo {
+    height: 40px;
+  }
+
+  &__link-logo {
+    text-decoration: none;
+    height: 40px;
+  }
+}
+
+.nav__container {
+  font-weight: 500;
+  text-transform: uppercase;
+  font-size: 12px;
+  text-decoration: none;
+  list-style-type: none;
+
+  .nav__container-list {
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    margin: 0;
+  }
+
+  .nav__container-link {
+    position: relative;
+    display: flex;
+    align-items: center;
+    padding-top: 23px;
+    padding-bottom: 22.5px;
+    color: $black-link;
```

Comment:
why do you have same styles for `is-active` class and `nav__container-link`? you should add `is-active` class only as modifier to general class and control only 'active' styles with `is-active`

Reply:
position: relative;
  color: $blue;
  padding: 23px 0;
 
because I can't remove padding and position from .is-active
everything will be crashed (npm test)

Reply:
is not fixed